docker-compose.yml

```diff
@@ -1,207 +1,257 @@
+networks:
+    go-quai_default:
+        external: true
 services:
   prime:
     environment:
       - TCP_PORT=30303
       - HTTP_PORT=8547
       - WS_PORT=8548
-      - NETWORK=garden
-      - SUB=--sub.urls ws://127.0.0.1:8579,ws://127.0.0.1:8581,ws://127.0.0.1:8583
+      - NETWORK=local
+      - SUB=--sub.urls ws://cyprus:8579,ws://paxos:8581,ws://hydra:8583
```

Comment:
These changes may be good, but why are these in this commit to change container names?